pkg/crdutil/crdutil.go

```diff
@@ -49,26 +51,32 @@ func (s *StringList) Set(value string) error {
 }
 
 var (
-	crdsDir StringList
+	files     []string
+	recursive bool
```

Comment:
Does it make sense to separate the handling of multiple non-recursive paths (including files) from the addition of recursive handling?

Reply:
AFAIK `kubectl` is also not separating it, so it is totally fine to keep it pretty open IMO. But also fine to me to split it somehow (you mean like: if multiple files are passed the recursive flag is not possible?)

Reply:
If the network-operator is not released with this change, better to just update as it will be first use of this package.

Reply:
I actually just meant as separate PRs. From https://github.com/NVIDIA/gpu-operator/pull/1137 it seems that we don't use recursive mode. Is it used in the network operator?

Reply:
No, it is not used by the network-operator. Maybe it makes sense to remove this new feature from the PR, you are right. The request adding it to this PR came through https://github.com/NVIDIA/k8s-operator-libs/pull/58#discussion_r1859239149 (at least that's how I understood it)